enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/provider/rest/ConsentTextRequestDto.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * PowerAuth Enrollment Server
+ * Copyright (C) 2022 Wultra s.r.o.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.wultra.app.onboardingserver.provider.rest;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+
+import java.util.Objects;
+
+/**
+ * @author Lubos Racansky, lubos.racansky@wultra.com
+ */
+@JsonTypeName("ConsentTextRequest")
+@javax.annotation.Generated(value = "org.openapitools.codegen.languages.JavaJAXRSSpecServerCodegen")
+class ConsentTextRequestDto {
+
+    private String processId;
+    private String userId;
+    private LanguagePropertyDto language;
+    private ConsentTypePropertyDto consentType;
+
+    /**
+     * Represents unique identifier of one specific mobile token reactivation flow
+     **/
+    public ConsentTextRequestDto processId(String processId) {
+        this.processId = processId;
+        return this;
+    }
+
+
+    @JsonProperty("processId")
+    public String getProcessId() {
+        return processId;
+    }
+
+    @JsonProperty("processId")
+    public void setProcessId(String processId) {
+        this.processId = processId;
+    }
+
+    /**
+     * Represents user identifier
+     **/
+    public ConsentTextRequestDto userId(String userId) {
+        this.userId = userId;
+        return this;
+    }
+
+
+    @JsonProperty("userId")
+    public String getUserId() {
+        return userId;
+    }
+
+    @JsonProperty("userId")
+    public void setUserId(String userId) {
+        this.userId = userId;
+    }
+
+    /**
+     *
+     **/
+    public ConsentTextRequestDto language(LanguagePropertyDto language) {
+        this.language = language;
+        return this;
+    }
+
+
+    @JsonProperty("language")
+    public LanguagePropertyDto getLanguage() {
+        return language;
+    }
+
+    @JsonProperty("language")
+    public void setLanguage(LanguagePropertyDto language) {
+        this.language = language;
+    }
+
+    /**
+     *
```

Comment:
Missing javadoc?

Reply:
dtto Make the code perfect or keep it raw? https://github.com/wultra/enrollment-server/pull/287/files#r932072572